test/rest-adapter.test.js

```diff
@@ -200,6 +200,32 @@ describe('RestAdapter', function() {
       });
     });
 
+    describe('getArgByName()', function() {
+      var anArg = [{
+        arg: 'argName1',
+        type: String,
+      }, {
+        arg: 'argName2',
+        type: String,
+      }];
```

Comment:
I reformatted the code to follow the convention we are using in LoopBack, where each arg definition is on a single line.

```
[
 {arg: 'argName1', /*...*/},
 {arg: 'argName2', /*...*/},
]
```

I have also renamed the variable to make the intent more clear.